git_machete/cmd.py

```diff
@@ -149,7 +151,16 @@ def __init__(self) -> None:
         self.opt_yes: bool = False
 
 
-def ask_if(cli_ctxt: CommandLineContext, msg: str, opt_yes_msg: Optional[str], apply_fmt: bool = True) -> str:
+def ask_if(
+    cli_ctxt: CommandLineContext,
+    msg: str,
+    opt_yes_msg: Optional[str],
+    apply_fmt: bool = True,
+    override: Optional[str] = None,
+) -> str:
+    if override:
+        print((fmt(msg) if apply_fmt else msg) + f" {override} [override]")
```

Comment:
Hmmm do we actually want to print anything here? I'd think just skipping the message should be fine